pages/addCourseForm.js

```diff
@@ -51,11 +63,10 @@ export default function addCourseForm() {
 
     } else {
       setHasBeenUpdated(!hasBeenUpdated)
-      const newCourseIndex = sharedState.courses.length - 1
-      const newCourseId = sharedState.courses[newCourseIndex].id
-      setSharedState({...sharedState, currentCourse: newCourseId})
+      router.push('/courseDashboard')
```

Comment:
This is a great solution for routing to the correct page after the course is created!